torch/optim/lr_scheduler.py

```diff
@@ -929,7 +931,7 @@ def _reduce_lr(self, epoch):
             if old_lr - new_lr > self.eps:
                 param_group['lr'] = new_lr
                 if self.verbose:
-                    print('Epoch {:5d}: reducing learning rate'
+                    print('Epoch {:5f}: reducing learning rate'
```

Comment:
Thanks for the fix! This one needs to be updated the same way right?

Reply:
Done